tests/periph_flashsector/main.c

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright (C) 2017 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Manual test application for sectored Flash peripheral driver
```

Comment:
I think this test can be abstracted a bit more and be merged with the flashpage test. Something like tests/periph_flash could take into account all possible uses of internal flash.